source/EduCATS/Pages/Login/Views/LoginPageView.cs

```diff
@@ -199,6 +202,23 @@ Entry createPasswordEntry(Style style)
 			return password;
 		}
 
+		Button createParentalButton()
+		{
+			var parentalButton = new Button
+			{
+				Text = "Контроль успеваемости",
```

Comment:
Should be a constant (language based). E.g. `CrossLocalization.Translate("test_passing_answer")`